src/pages/index.js

```diff
@@ -93,28 +220,52 @@ addButton.addEventListener("click", handleOpenAddImageModal);
 /* -------------------------------------------------------------------------- */
 
 const previewImagePopup = new PopupWithImage("#preview-image-modal");
+
+const deleteImageConfirmPopup = new PopupWithFormConfirmDelete(
+  "#confirm-modal",
+  handleDeleteImageSubmit
+);
+
+function handleDeleteImageSubmit(data) {
+  api.removeImageFromAPI("cards", data.imageId);
+  let cardToDelete = document.getElementById(data.imageId);
```

Comment:
Constant variables should be declared using `const` keyword

Reply:
I couldn't change this to const without having errors.

Uncaught TypeError: Assignment to constant variable.
    at PopupWithFormConfirmDelete.handleDeleteImageSubmit [as _handleFormSubmit] (index.js:233:15)
    at HTMLDivElement._handleSubmit (PopupWithFormConfirmDelete.js:22:10)

Reply:
This is because you need to remove the card element using card class methods (Don't remove the element in index.js)

Reply:
I think I got this since I rewrote this function. Thanks!